### ISSUE TITLE
Release the Pretraining Datasets

### ISSUE THREAD

**Issue by YaooXu** [kanishkg/cognitive-behaviors#4]
Thanks for your excellent work! 

I am interested in conducting further research and analysis on the continued pretraining dataset you constructed from OpenWebMath, such as the specific sample size and length distribution.  Could you please release this dataset on Huggingface? Thank you very much!

**Comment by joshbarua** [comment]
^ thank you! Unless [obiwan96/owm_nonev4_sft](https://huggingface.co/datasets/obiwan96/owm_nonev4_sft/viewer/default/train?row=0&views%5B%5D=train) is the dataset you used. The basic hyperparameters for sft would also be appreciated!

**Comment by kanishkg** [comment]
That wasn't the final version of the dataset! I will upload a collection with the sft'ed model, datasets soon (over the coming week)! Just preparing the project for COLM right now!

**Comment by kanishkg** [comment]
Hi here are the datasets:
1. Behaviors: https://huggingface.co/datasets/obiwan96/owm-cog-behaviors with sft ckpt at: https://huggingface.co/obiwan96/owmqa_method
2. No behaviors: https://huggingface.co/datasets/obiwan96/owm-no-behaviors with sft ckpt at: https://huggingface.co/obiwan96/owmqa_none

PS: the dataset for behaviors can probably be made a little more clean by iterating on the prompts!

**Comment by snat1505027** [comment]
Hi @kanishkg, thanks for releasing the dataset! I could not find the cognitive behavior labels in the dataset. Is there a way to determine this correctly? I think the script shared in the readme (pretraining_analysis/relabel_pretrain_offline.py) might serve this purpose, but would be great if you already have that labeling in the dataset. Thank you!
